### ISSUE TITLE
Buttons do not have an accessible name

### ISSUE THREAD

**Issue by akomiqaia** [fac19/week2-guardians#64]
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.

Failing Elements

`button#movie-submit.movie-form__submit`
